docs/ballerina-by-example/examples/fork-join-variable-access/fork-join-variable-access.bal

```diff
@@ -1,87 +1,83 @@
 import ballerina/io;
 
-@Description {value:"In scope variables can be accessed within workers of fork-join statement."}
+@Description {value:"The in scope variables can be accessed by the workers of in the fork-join statement."}
 function main (string[] args) {
-    // Define variables which are visible to the forked workers.
+    // These variables can be accessed by the forked workers.
     int i = 100;
     string s = "WSO2";
     map m = {"name":"Abhaya", "era":"Anuradhapura"};
 
     // Declare the fork-join statement.
     fork {
         worker W1 {
-            // Change the value of the integer variable "i" within the worker W1.
+            // Change the value of the Integer variable 'i' within the worker W1.
             i = 23;
-            // Change the value of map variable "m" within the worker W1.
+            // Change the value of map variable 'm' within the worker W1.
             m["name"] = "Rajasinghe";
-            // Define a variable within the worker to send to join block.
+            // Define a new variable within the worker to send to the join block.
             string n = "Colombo";
-            // Send data to join block of the fork-join from worker W1.
+            // Send the data to the join block of the fork-join from worker W1.
             i, n -> fork;
         }
 
         worker W2 {
-            // Change the value of string variable "s" within the worker W2.
+            // Change the value of String variable 's' within the worker W2.
             s = "Ballerina";
-            // Change the value of map variable "m" within the worker W2.
+            // Change the value of map variable 'm' within the worker W2.
             m["era"] = "Kandy";
-            // Send data to join block of the fork-join from worker W2.
+            // Send the data to the join block of the fork-join from worker W2.
             s -> fork;
         }
     } join (all) (map results) {
 
         any[] r1;
         any[] r2;
-        // Declare variables to receive the results from forked workers W1 and W2.
-        // The 'results' map contains a map of any type array from each worker
-        // defined within the fork-join statement.
-        // Values received from worker W1 are assigned to any array of r1.
+        // Declare variables to receive the results from the forked workers W1 and W2.
+        // The 'results' map contains a map of the results from the workers within the fork-join statement in the
+        // 'any' type array.
```

Comment:
"map of any type array" vs "map of the results from the workers within the fork-join statement in the 'any' type array." has somewhat different meaning. This should be map of any type array..